File: frontend/Echange-J-L-Angular/src/app/models/Jouet.ts
import { CategorieJouet } from "./Categorie_jouet";
import { Etat } from "./Etat";
export class Jouet {

  id!: String;
  titre!: String;
  description!: String;
  etat_jouet!: Etat;
  categorie_jouet!: CategorieJouet;


}

File: backend/Spring/exchange/src/main/java/iset/project/exchange/entities/Jouet.java
package iset.project.exchange.entities;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.index.Indexed;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@Document("jouet")
public class Jouet {
    @Id
    private String id ;
    @Indexed(unique = true)
    private String titre ;
    private String description ;
    private Etat etat_jouet ;
    private CategorieJouet categorie_jouet ;
    private String uploaded_by;
}
